pypairtree/pairtreelist.py

```diff
@@ -33,9 +33,9 @@ def listIDs(basedir):
         for obj in objects:
             doi = os.path.split(obj)[1]
             # print with prefix and original chars in place
-            print prefix + pairtree.deSanitizeString(doi)
```

Comment:
What do you think about adding the `from __future__ import print_function` import to this file?

Reply:
Don't mind adding it. Doesn't hurt anything.